tests/integration_tests/tests/agentless_tests/test_manager_misc.py

```diff
@@ -87,8 +87,17 @@ def test_logrotation(self):
             'touch /var/log/cloudify/mgmtworker/logs/test.log')
 
         self.logger.info('Cancelling date suffix on rotation...')
-        sed_cmd = 'sed -i -e s/dateext/#dateext/ /etc/logrotate.conf'
-        self.execute_on_manager(sed_cmd)
+        # We need to do this separately for each logrotate configuration.
+        for logrotate_cfg in ['cloudify-amqp-postgres',
+                              'cloudify-mgmtworker',
+                              'cloudify-rabbitmq',
+                              'composer',
+                              'nginx',
+                              'restservice',
+                              'stage']:
+            self.logger.info('Cancelling for %s' % logrotate_cfg)
```

Comment:
in general we use `.format` instead of %-formatting
however python's logging can do this: `logger.info('Cancelling for %s', logrotate_cfg)` (so that the potentially-expensive string formatting isn't done unless the log is actually emitted, wrt. logging level etc)
doing %-formatting here is really the worst choice :P